app/controllers/posts_controller.rb

```diff
@@ -1,20 +1,12 @@
 class PostsController < ApplicationController
   def index
-    @current_user = current_user
     @user = User.find(params[:user_id])
-
-    redirect_to root_path if @user.nil?
-    @posts = @user.posts
+    @posts = Post.includes(:comments).where(author_id: @user.id)
   end
 
```

Comment:
![image](https://github.com/Microverse-Fullstack-Program/rails-blog-app-v2/assets/93039359/b505aa83-1235-4a50-aab9-5f9cced8d0cb)

**[OPTIONAL]**

- [ ] It would be nice if you put the right name of your app in the readme file to make your descriptive and professional 😊